test/onnx/test_utility_funs.py

```diff
@@ -1044,6 +1044,137 @@ def forward(self, input, other):
         iter = graph.nodes()
         self.assertEqual(next(iter).kind(), "custom_namespace::custom_op")
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_autograd_layernorm_shape(self):
+        def symbolic_pythonop(ctx: torch.onnx.SymbolicContext, g, *args, **kwargs):
+            return g.op("com.microsoft::PythonOp")
+
+        register_custom_op_symbolic("prim::PythonOp", symbolic_pythonop, 1)
+        self.addCleanup(unregister_custom_op_symbolic, "prim::PythonOp", 1)
+
+        # necessay parameters for transformer embeddings
+        hidden_size, vocab_size, max_position_embeddings, type_vocab_siz, batch_size = (
+            768,
+            128000,
+            512,
+            0,
+            16,
+        )
+
+        # wrap nn.layernorm into autograd.func repro the same
+        # issue as using apex fusedlayernorm
+        class CustomLayerNorm(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, embedding):
+                ctx.save_for_backward(embedding)
+                layer_norm = torch.nn.LayerNorm(hidden_size, eps=1e-12)
```

Comment:
this uses some random affine weights that are then discarded?

Reply:
This test is meant to repro a onnx export failure, and this setting is enough for it.